app/controllers/events_controller.rb

```diff
@@ -8,6 +8,10 @@ def index
   end
 
   def show
+    #Fetching all users instead of users just in campus
```

Comment:
If we changed this comment to explain why that needs to change and started the comment with `TODO:` it would make it more actionable and anyone could run `rake notes` to see any outside tasks.